tests/test_pytorch_data_teacher.py

```diff
@@ -130,7 +130,11 @@ def get_teacher_act(defaults, teacher_processed=False, agent_to=None):
             val1 = agent_processed_observation[key]
             val2 = teacher_processed_act[key]
             if type(val1) is torch.Tensor:
-                self.assertTrue(bool(torch.all(torch.eq(val1, val2))))
+                self.assertTrue(bool(torch.all(torch.eq(val1, val2))),
```

Comment:
Should this be torch.allclose instead? I don't know what the type of var1, var2 are off the top of my head.

Reply:
Ahh that's a useful function, but in this case val1 and val2 are LongTensors so I don't think it is necessary